engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/EditorExtractor.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.engine.definition
+
+import java.lang.reflect.Parameter
+
+import pl.touk.nussknacker.engine.api.definition._
+import pl.touk.nussknacker.engine.api.editor.{DualEditor, RawEditor, SimpleEditor}
+
+object EditorExtractor {
+
+  def extract(param: Parameter): Option[ParameterEditor] = {
+    val dualEditorAnnotation = param.getAnnotation(classOf[DualEditor])
+    val simpleEditorAnnotation = param.getAnnotation(classOf[SimpleEditor])
+    val rawEditorAnnotation = param.getAnnotation(classOf[RawEditor])
+    if (dualEditorAnnotation != null)
```

Comment:
You can use there match pattern it will look better :)

Reply:
Done